poh/src/poh_recorder.rs

```diff
@@ -284,7 +284,7 @@ pub struct PohRecorder {
     start_bank_active_descendants: Vec<Slot>,
     start_tick_height: u64, // first tick_height this recorder will observe
     tick_cache: Vec<(Entry, u64)>, // cache of entry and its tick_height
-    working_bank: Option<WorkingBank>,
+    pub working_bank: Option<WorkingBank>,
```

Comment:
don't think we should make this `pub` just for benches - see my other comment about only needing one sanity check assert